src/Main/Gear.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { getPlayerName } from 'selectors/url/report';
+
+class Gear extends React.PureComponent {
+  static propTypes = {
+    combatants: PropTypes.object.isRequired,
+    playerName: PropTypes.string.isRequired,
+  };
+  render() {
+    const playersList = this.props.combatants.players;
+    const iconBaseUrl = 'http://render-us.worldofwarcraft.com/icons/56/';
+    const toolTipBaseUrl='http://www.wowhead.com/item=';
+    let currentPlayer;
+    for (const prop in playersList) {
+      if (playersList[prop]._combatantInfo.name === this.props.playerName) {
+        currentPlayer=  playersList[prop]._combatantInfo;
```

Comment:
You can get the current player with `this.props.combatants.selected`. Also I recommend changing the prop from taking the `combatants` prop to just `combatant` or `selectedCombatant` or something similar. That way you don't need the playerName.

Reply:
Done, Gear now takes selectedCombatant  as the only prop.